lib/travis/logs/app.rb

```diff
@@ -167,11 +174,54 @@ def initialize(auth_token: ENV['AUTH_TOKEN'].to_s,
         status 204
       end
 
-      put '/logs/:job_id/archived' do
+      put '/logs/:log_id/archived' do
```

Comment:
Is there a reason for this endpoint using log ID, but the other ones using job ID? Seems like that could lead to some confusion if `PUT /logs/123` and `PUT /logs/123/archived` modifies different logs.